django/forms/templates/django/forms/attrs.html

```diff
@@ -0,0 +1 @@
+{% for name, value in attrs.items %}{% if value is not False %} {{ name }}{% if value is not True %}="{{ value|stringformat:'s' }}"{% endif %}{% endif %}{% endfor %}
```

Comment:
I think I need a bit of help here. 

I did this as it mirrors the widget template, also see https://github.com/django/django/pull/8639 where this was introduced. 

I _could_ introduce a similar test but https://github.com/django/django/pull/14519/ is removing the setting used to demonstrate this test. I'm therefore if it's that the test is no longer valid (i.e. the test needs updating to show a different localisation issue) or if we no longer need this as folk can't configure their project to come accross this issue going forward?

Reply:
Of course I am just taking time to understand this. It's just the setting that is being removed and going forward it will be localised by default. 

I'll add tests for this!

Reply:
So it sounds like we need to keep it in. Perfect - a test will make it clear should someone try to remove it in the future!